### PR TITLE
Sslfactory support

### PR DESCRIPTION
**Pull request by kpanwar**
Sslfactory method name

### CODE REVIEW

File: src/test/java/com/microsoft/aad/adal4j/AuthenticationContextTest.java [hunk 1]
```diff
@@ -164,79 +159,6 @@ public void testAcquireTokenAuthCode_KeyCredential() throws Exception {
         PowerMock.resetAll(ctx);
     }
```
Comment (weijjia) [inline src/test/java/com/microsoft/aad/adal4j/AuthenticationContextTest.java]: why did you remove these tests?

Comment (kpanwar) [inline src/test/java/com/microsoft/aad/adal4j/AuthenticationContextTest.java]: one of the tests is a duplicate. Fixed the other one.

**Comment by weijjia** [comment]
:shipit:
